api/types/events/events.proto

```diff
@@ -762,9 +762,8 @@ message AccessRequestCreate {
     // variant).
     string ProposedState = 11 [ (gogoproto.jsontag) = "proposed_state,omitempty" ];
 
-    // RequestedResourceIDs is a list of resources to which access is being
-    // requested.
-    repeated string RequestedResourceIDs = 12 [ (gogoproto.jsontag) = "resource_ids,omitempty" ];
+    // RequestedResourceIDs is the set of resources to which access is being requested.
+    string RequestedResourceIDs = 12 [ (gogoproto.jsontag) = "requested_resource_ids,omitempty" ];
```

Comment:
Why don't we reference the protobuf type here instead of using embedded json?

Reply:
I brought this up on slack, ended up duplicating the type in the events package https://gravitational.slack.com/archives/C0DF0TPMY/p1652286139700439